README.md

```diff
@@ -65,13 +162,113 @@ python
+  for domain experts that is native to science and engineering.
+
+#### Domain Specific Packages
+
+The following are packages dedicated for domain experts of specific communities catering
+to their unique exploration needs.
+
+- [Earth-2 Studio](https://github.com/NVIDIA/earth2studio): Open source project
+  to enable climate researchers and scientists to explore and experiment with
+  AI models for weather and climate.
+
+#### Research packages
+
+The following are research packages that get packaged into Modulus once they are stable.
+
+- [Modulus Makani](https://github.com/NVIDIA/modulus-makani): Experimental library
+  designed to enable the research and development of machine-learning based weather and
+  climate models.
+- [Earth2 Grid](https://github.com/NVlabs/earth2grid): Experimental library with
+  utilities for working geographic data defined on various grids.
+- [Earth-2 MIP](https://github.com/NVIDIA/earth2mip): Experimental library with
+  utilities for model intercomparison for weather and climate models.
+
+## Who is using and contributing to Modulus
+
+Modulus is an open source project and gets contributions from researchers in the SciML and
+AI4science fields. While Modulus team works on optimizing the underlying SW stack, the
+community collaborates and contributes model architectures, datasets, and reference
+applications so we can innovate in the pursuit of developing generalizable model
+architectures and algorithms.
+
+Some latest examples of community contributors are [HP Labs 3D Printing team](https://developer.nvidia.com/blog/spotlight-hp-3d-printing-and-nvidia-modulus-collaborate-on-open-source-manufacturing-digital-twin/),
+[Stanford Cardiovascular research team](https://developer.nvidia.com/blog/enabling-greater-patient-specific-cardiovascular-care-with-ai-surrogates/),
+[UIUC team](https://github.com/NVIDIA/modulus/tree/main/examples/cfd/mhd_pino),
+[CMU team](https://github.com/NVIDIA/modulus/tree/main/examples/generative/diffusion) etc.
+
+Latest examples of research teams using Modulus are
+[ORNL team](https://arxiv.org/abs/2404.05768),
+[TU Munich CFD team](https://www.nvidia.com/en-us/on-demand/session/gtc24-s62237/) etc.
+
+Please navigate to this page for a complete list of research work leveraging Modulus.
+For a list of enterprises using Modulus refer [here](https://developer.nvidia.com/modulus).
+
+Using Modulus and interested in showcasing your work on
+[NVIDIA Blogs](https://developer.nvidia.com/blog/category/simulation-modeling-design/)?
+Fill out this [proposal form](https://forms.gle/XsBdWp3ji67yZAUF7) and we will get back
+to you!
+
+## Why are they using Modulus
+
+Here are some of the key benefits of Modulus for SciML model development:
+
+<!-- markdownlint-disable -->
+<img src="docs/img/Value%20prop/benchmarking.svg" width="100"> | <img src="docs/img/Value%20prop/recipe.svg" width="100"> | <img src="docs/img/Value%20prop/performance.svg" width="100">
```

Comment:
These paths should be fixed. The icons don't show up in its current form.

Reply:
Great catch! I forgot to update these. Will change.